packages/material-ui-unstyled/src/BadgeUnstyled/BadgeUnstyled.test.js

```diff
@@ -0,0 +1,199 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { createMount, createClientRender, describeConformance } from 'test/utils';
+import BadgeUnstyled from './BadgeUnstyled';
+import classes from './badgeClasses';
+
+function findBadge(container) {
+  return container.firstChild.querySelector('span');
+}
+
+describe('<BadgeUnstyled />', () => {
```

Comment:
Do we need to test the unstyled components? How about we only test the styled version?

Reply:
I had the same question but was thinking rather whether we should test only the Unstyled, as then those are used in the core.. But in the end, I decided that maybe it is better to test both.. Not sure, what do you think? If we test only the core, we won't have any tests for the unstyled...

Reply:
I definitely think that we shouldn't have duplicated tests. For the Autocomplete, we barely test useAutocomplete, only for specific aspects of the hook, the approach seems to have been working well. I haven't looked at the content of the tests.

How about we approach the problem from a coverage perspective? Only include tests that increase coverage, remove anything else.

Reply:
Makes sense, will do it (probably is enough to have only a conformant test for the unstyled...)

Reply:
Done

Reply:
describe conformance sounds great, for a smoke test. I guess we will find more cases in the future where it makes sense.